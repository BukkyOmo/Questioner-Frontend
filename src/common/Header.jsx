import React from 'react';
import { NavLink } from 'react-router-dom';

const Header = () => (
  <nav>

    <ul>
      <li id='logo'><NavLink to='/'>Questioner</NavLink></li>

      <li><a href='/signup'>Sign Up</a></li>

      <li><a href='/login'>Sign In</a></li>

    </ul>

  </nav>
);

export default Header;
