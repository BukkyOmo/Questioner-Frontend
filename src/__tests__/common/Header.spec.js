import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../common/Header';

describe('<Header />', () => {
  it('renders h1 element', () => {
    const wrapper = shallow(<Header />)
    expect(wrapper.find('ul').exists()).toBe(true);
  });
   it('renders correctly using snapshot', () => {
    expect(Header).toMatchSnapshot();
   });
});
