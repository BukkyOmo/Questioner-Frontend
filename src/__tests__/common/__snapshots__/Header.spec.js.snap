// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders correctly using snapshot 1`] = `[Function]`;
